front/app/containers/Admin/formBuilder/components/FormBuilderSettings/index.tsx

```diff
@@ -1,148 +1,119 @@
-
-  let translatedStringKey: ReactIntl.FormattedMessage.MessageDescriptor | null =
-    null;
-  if (field.input_type === 'text') {
-    translatedStringKey = messages.shortAnswer;
+const FormBuilderSettings = ({ field, onDelete, onClose }: Props) => {
+  const locales = useAppConfigurationLocales();
+  let translatedStringKey: MessageDescriptor | null = null;
+  switch (field.input_type) {
+    case 'text':
+      translatedStringKey = messages.shortAnswer;
+      break;
+    case 'multiselect':
+      translatedStringKey = messages.multipleChoice;
+      break;
   }
 
-  const onStateChange = (key: string, value: Multiloc | boolean) => {
-    setFieldState({
-      ...fieldState,
-      [key]: value,
-    });
-  };
-
-  const { isRequired, questionTitle, questionDescription } = fieldState;
-
-  return (
-    <Box
-      position="fixed"
-      right="0"
-      top={`${stylingConsts.menuHeight}px`}
-      zIndex="99999"
-      px="20px"
-      w="400px"
-      h="100%"
-      background="white"
-      boxShadow="-2px 0px 1px 0px rgba(0, 0, 0, 0.06)"
-    >
-      <Box position="absolute" right="8px" mb="20px">
-        <CloseIconButton
-          a11y_buttonActionMessage={messages.close}
-          onClick={onClose}
-          iconColor={colors.label}
-          iconColorOnHover={'#000'}
-        />
-      </Box>
-      {translatedStringKey && (
-        <SectionTitle>
-          <FormattedMessage {...translatedStringKey} />
-        </SectionTitle>
-      )}
-      <SectionField>
-        <InputMultilocWithLocaleSwitcher
-          type="text"
-          label={<FormattedMessage {...messages.questionTitle} />}
-          valueMultiloc={questionTitle}
-          onChange={(value: Multiloc) => onStateChange('questionTitle', value)}
-        />
-      </SectionField>
-      <SectionField>
-        <InputMultilocWithLocaleSwitcher
-          type="text"
-          label={<FormattedMessage {...messages.questionDescription} />}
-          valueMultiloc={questionDescription}
-          onChange={(value: Multiloc) =>
-            onStateChange('questionDescription', value)
-          }
-        />
-      </SectionField>
-      <SectionField>
-        <Toggle
-          checked={!!isRequired}
-          onChange={() => onStateChange('isRequired', !isRequired)}
-          label={
-            <Text as="span" color="adminTextColor" variant="bodyM" my="0px">
-              <FormattedMessage {...messages.required} />
-            </Text>
-          }
-        />
-      </SectionField>
-      <Box display="flex" justifyContent="space-between">
-        <Button
-          icon="delete"
-          buttonStyle="primary-outlined"
-          borderColor={colors.red500}
-          textColor={colors.red500}
-          iconColor={colors.red500}
-          onClick={() => onDelete(field.id)}
-          minWidth="160px"
-        >
-          <FormattedMessage {...messages.delete} />
-        </Button>
+  if (!isNilOrError(locales) && locales !== undefined) {
```

Comment:
Yes, this part of the work inherited from Amanda's last PR from Friday. I made a note fix this separately along with some changes since she is away and couldn't make those changes today